### ISSUE TITLE
Mark dominator edges in CFG

### ISSUE THREAD

**Issue by yugr** [Kazhuu/asm2cfg#24]
Knowing that one block dominates another may be useful for reading the CFG. Such edges can be highlighted with different color or thickness.

**Comment by x213212** [comment]
hi, I have slightly modified this project 
https://github.com/x213212/asm2cfg
so that it can read riscv asm to draw the control flow graph, and the latter added the tarjan algorithm according to your suggestion so that it can find the loops in asm

![newmatrix_test](https://user-images.githubusercontent.com/9297254/203045735-2288769a-2ec3-4d52-97d4-d15576d18526.svg)

**Comment by Kazhuu** [comment]
Yeah this looks very promising. Can you make a PR about these so I can review them?
